users/README.md

```diff
@@ -303,3 +304,88 @@ Updates roles for the User.
   - **Code:** 500
     - **Content:**
       `{ 'statusCode': 500, 'error': 'Internal Server Error', 'message': 'An internal server error occurred' }`
+
+## PATCH /users
+
+Archive users if not in Discord.
+
+- **Params**  
+  None
+- **Query**
+  - Optional: `debug=[boolean]`(`debug` when set to true returns additional result in response to help debugging)
+- **Headers**  
+  Content-Type: application/json
+- **Cookie**  
+  rds-session: `<SUPERUSER JWT>`
+- **Body**
+
+  ```json
+  {
+    // Defines action associated with PATCH /users route. Calls specific api based on action passed. If wrong payload is passed it throws an error
```

Comment:
inside a JSON we cant add comments hence this is highlighted in red. please move it above from where json starts